app/config/config.yml

```diff
@@ -68,3 +75,9 @@ swiftmailer:
     username: '%mailer_user%'
     password: '%mailer_password%'
     spool: { type: memory }
+
+assetic:
+    debug: '%kernel.debug%'
+    use_controller: '%kernel.debug%'
+    filters:
+        cssrewrite: ~
```

Comment:
Sure about adding assetic? This could/should have some consequences to deployments etc as well. But maybe you simply don't use it (yet)?

Reply:
Most SURFnet Symfony projects use Assetic. Using it should not pose any serious problems as these have been solved before.